lib/component.ex

```diff
@@ -92,4 +92,47 @@ defmodule LiveSvelte do
   end
 
   defp id(name), do: "#{name}-#{System.unique_integer([:positive])}"
+
+  @doc false
+  def get_props(assigns) do
+    prop_keys =
+      assigns
+      |> Map.get(:__changed__)
+      |> Map.keys()
+
+    assigns
+    |> Enum.filter(fn
+      {:svelte_opts, _v} -> false
+      {k, _v} -> k in prop_keys
+    end)
+    |> Enum.into(%{})
+  end
+
+  @doc false
+  def get_ssr(assigns) do
+    case get_in(assigns, [:svelte_opts, :ssr]) do
+      nil -> true
+      ssr -> ssr
+    end
```

Comment:
there is no default option in Kernel.get_in but this is also an option
```suggestion
    get_in(assigns, [:svelte_opts, :ssr]) || true
    end
```

Reply:
sorry - this won't work